config/manager/kustomization.yaml

```diff
@@ -1,2 +1,7 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/vedantm/airflow-operator
```

Comment:
@kaxil  - This would benefit from a recommended docker registry for hosting airflow related docker images.

Reply:
Ah yes this shouldn't be in the PR, my bad.